### ISSUE TITLE
Browser goes back to first slide after each export/refresh

### ISSUE THREAD

**Issue by fmind** [yjwen/org-reveal#316]
In a typical work flow, I export my slides to HTML and refresh the browser to see the changes.

But after each export/refresh, the browser goes back to the first slide (even if I were to slide 5 for instance).

I notice that the anchor of each slide changes after each export. They seem to be auto-generated by org-mode: `#/slide-org0be7ffd`

Is there a way to control this behaviour ?

**Comment by joonro** [comment]
This is a known issue and there is a pull request which deals with it (#284). You might want to try his forked version if you want to get this functionality.

**Comment by yjwen** [comment]
reveal.js provides consistent link like

> file:///home/yujie/tmp/tmp.html#/2/1

to the first slide of the second vertical slide group.

**Comment by Cadair** [comment]
Sorry to necrobump this thread, but it seems better than opening a new issue. I am still running into this.

I have spent 10 minutes chasing through the various issues and PR and I have tried with and without `+#OPTIONS: num:nil` and with and without `hash: true` in the reveal init options, and everything I try leads me to still getting slide IDs of the form `slide-org<uuid>` which means when I re-export and reload the talk it goes back to the start which is just infuriating. Any help would be greatly appreciated.
